static/index.html

```diff
@@ -172,40 +172,66 @@ <h4>Current Rules:</h4>
       <img src="/static/images/Discord-Logo-Black.png" class='discord-image' width="20px">
     or simply say hello to the project's creator <a href="mailto:tooangel@tooangel.de">Tobias Wilken</a>
 
-    <h2>Phase Two: Transparent Services</h2>
-    <p>Having clear and automated rules for progressing in open source projects is just the start.
-    Due to the DevOps (or GitOps) ideas, many operational tasks are now stored in repositories
-    and applied upon merge. So the challenge now is building (world driven) services that operate
-    on these ideas and are only maintained via Pull Requests. This provides a new level of trust
-    and a new entity <em>unavailable system data</em> (still searching for a better name).
-    Passwords, service credentials and the like is data which does not necessarily have to be
-    accessible by humans; it can therefore be maintained and used exclusively in the system itself.</p>
-
-    <p>For example: In one of the companies I was working for, the employees were using
-    Chromeboxes. As part of their daily work they had to split and merge PDFs.
-    There are a lot of (free) services out there that can do the job but
-    they can't be used due to privacy concerns - What happens with my PDFs? Do they store them? Do
-    they use them? - I hope (and guess) that they don't but I can't be sure.</p>
-
-    <p>Compare that to a PDF service that is open source, world driven and uses continuous deployment.
-    No one has access to any credentials and changes can only be done via public Pull Requests -
-    That sounds far more secure to me.</p>
-
-
-    <h2>Phase Three: Down the Rabbit Hole</h2>
-    <p>In my mind these examples use:</p>
-    <ul>
-      <li>Some CI services for testing and deployment</li>
-      <li>Some automated hosting environment (PaaS)</li>
-      <li>Some databases...</li>
-    </ul>
-    <p>The web service itself is world driven but can I trust the other parties?
-    Why not build the infrastructure in the same way? Using IaaS instead of
-    higher level managed server is a lot to do but it is doable (I know, I've already built a PaaS).
-    World driven fully automated IaaS, IMHO, is more challenging.</p>
+    <h3>A Speculative Future...</h3>
+    <p>The further phases are only speculative and lay blurry upon the project's horizon. Scroll
+    further and our current ideas will be clarified, join us and you can help shape them.</p>
+
+    <script src="https://code.jquery.com/jquery-3.5.1.min.js"
```

Comment:
I don't think it is a good idea or good style to include scripts somewhere from the internet. So if including a library we should add it to the `package.json` and provide via the webserver.

Besides that, we are using react for the rest of the page, so I think we should stick with that. I was already thinking about how to include it in the index page, but I wouldn't want to move the content from here into a `.js` file.

Reply:
Good points. I'll be honest, I was just thinking about the effect and how I could do it rather than the best way to do it for this project.

Not sure where to go from here though. Using react makes sense but I agree moving the content to a .js doesn't seem right.

Reply:
I think it is a good approach, especially to align what we both have in mind and I think we are aligned now.

I think we can have this PR, without the javascript and set the opacity values in css to the max values from the js code.
Get this merged and see how we can continue.

After that, I'm not sure :-)

What I have in mind, but do not have a solution:
- E.g. the footer could be shared between the front page and the dashboard. So I'm thinking about having some kind of build step, which generates the index.html out of some templates (e.g. header, content, footer)
- I think you can't easily change css content from react, so the approach would be setting classes. If we have the content split into multiple files, we could use the front page content in the index.html and in the react component, too.
This way react will not change the content when rendering, but we can play with classes.
- I couldn't find a way (yet), how to read the existing content of a DOM node and go from there with react. (I was looking into the react-server concept, maybe there we could find some ideas).

Or anything else I didn't think of yet :-)

Reply:
So set the opacity at like 0.7~ and call it a day? I'm sure I could find a js, as opposed to jquery, way to make the effect and that would remove the need for a jquery import. Would something like that work?

Reply:
Both fine.
I guess there should be a simple way to solve it in vanilla javascript. I'm totally fine with that.